src/apps/statements/tests/getStatements/utils/assertStatementsPageResultsAndOrder.ts

```diff
@@ -0,0 +1,60 @@
+import assert from 'assert';
+import { isArray, isString } from 'lodash';
+
+import ClientModel from '../../../models/ClientModel';
+import Service from '../../../serviceFactory/Service';
+
+/* istanbul ignore next */
```

Comment:
I don't think we need this ignore any more do we?

Reply:
@ryansmith94 , Without this I have next
![Screenshot 2020-03-16 at 14 37 59](https://user-images.githubusercontent.com/6347345/76759145-c7503f80-6793-11ea-8b46-c8b8b3988c8a.png)